constants/creator_groups.json

```diff
@@ -343,7 +343,8 @@
         "Salesforce Research",
         "Samsung R&D Institute",
         "Tencent AI Lab",
-        "iFLYTEK Research"
+        "iFLYTEK Research",
```

Comment:
Could you put these in alphabetical order -- not sure why iFLYTEK was out of order. Thanks!

Reply:
Fixed in latest commit.